src/pihole-updatelists.conf

```diff
@@ -72,3 +72,9 @@ REGEX_BLACKLIST_URL=""
 
 ; Branch to pull remote checksum and update from
 ;GIT_BRANCH="master"
+
+; Crontab expression to automatically update lists and gravity
```

Comment:
This is completely useless for non-docker installations and should not be in config by default, could it be moved to `60-pihole-updatelists.sh` perhaps?